js/cell.js

```diff
@@ -24,10 +30,12 @@ define([
     };
 
     domReady(function () {
+        initializeURLs();
+
         var style = document.createElement("style");
         style.innerHTML = css.replace(
```

Comment:
The sagecell URLs are read from the DOM (extracted from the `<script>` tag, etc.). This cannot be done until the DOM loads. However, all the current code assumes that the URLs are preset on a global object. This is impossible to do. However, the URLs are never used until after a call to `cell.js`, so it's safe to initialize them there.

The old version worked because no code was executed until `domReady` finished.